tools/shared.py

```diff
@@ -288,8 +288,14 @@ def get_clang_version():
 
 def check_llvm_version():
   actual = get_clang_version()
-  if EXPECTED_LLVM_VERSION in actual:
+  if actual.startswith('%d.' % EXPECTED_LLVM_VERSION):
     return True
+  # When running in CI environment we also silently allow the next major
+  # version of LLVM here so that new versions of LLVM can be rolled in
+  # without disruption.
+  if 'BUILDBOT_BUILDNUMBER' in os.environ:
```

Comment:
Is `BUILDBOT_BUILDNUMBER` set in Circle or just in Chromium?

Reply:
Only in chromium CI, but that is the case I think we care about.  Once the change rolls in which can create an emscripten-side PR to update the expected version.    The thing I'm trying to avoid is having to create a temporary PR in emscripten than then gets reverted later.  

I think its fine if emscripten PRs start going red after a clang version bump.. as a long as the fix is simple.    This is the same behaviour we have for code size regressions.. they roll silently in chromium CI and show up as failures that needs attention in the emscripten CI.